certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/model/CertificateActionFactory.java

```diff
@@ -143,6 +143,14 @@ public static CertificateAction create(CertificateActionSpecification actionSpec
               )
           )
           .build();
+      case SEND_AFTER_SIGN -> CertificateActionSendAfterSign.builder()
+          .certificateActionSpecification(actionSpecification)
+          .actionRules(
```

Comment:
Do we need to validate that it has not already been sent?

Reply:
Even tho I dont think its a scenario that can occur, I think you are correct and we should do this check! Good point!